src/benchmark/table_generator.hpp

```diff
@@ -62,7 +62,8 @@ class TableGenerator {
 
   std::shared_ptr<Table> generate_table(const std::vector<ColumnDataDistribution>& column_data_distributions,
                                         const size_t num_rows, const size_t chunk_size,
-                                        std::optional<EncodingType> encoding_type = std::nullopt);
+                                        std::optional<EncodingType> encoding_type = std::nullopt,
+                                        bool use_multiple_partitions = false);
```

Comment:
Maybe call this `use_roundrobin_placement`, `numa_roundrobin_placement`, or something like that?

Reply:
The roundrobin placement is currently an implementation detail that is not really important to the user of the table generator. There are no plans for different placements. This switch is about whether we place on different NUMA Partitions at all or whether we let them be default placed.

Reply:
Ah I see what you are saying. One thing that I think is important though is making it clear that we are talking about whole `chunk` placement instead of `pages`. The normal `interleave` policy e.g. uses multiple partitions as well, but distributes on a `page` level.